models/classes/class.QtiTestService.php

```diff
@@ -96,6 +96,9 @@ class taoQtiTest_models_classes_QtiTestService extends TestService
     public const XML_TEST_PART = 'testPart';
     public const XML_ASSESSMENT_SECTION = 'assessmentSection';
     public const XML_ASSESSMENT_ITEM_REF = 'assessmentItemRef';
+
+    private const IN_PROGRESS_LABEL = 'in progress';
```

Comment:
I think LABEL_IN_PROGESS is better, because then you would just add another LABEL_ and everything would be in same code block

Reply:
The text for the label when status is in progress: 'in progress' <del>status</del> label `IN_PROGRESS_LABEL`
For me sounds more clear than: label <del>for</del> 'in progress' <del>status</del> - much more context dropped